### DISCUSSION TITLE
when I open vim then I installed ale,  they tips me E121: undefind var: g:ale root

### DISCUSSION THREAD

**Discussion by Alice-pleasance-liddell** [w0rp/ale#3675] [Q&A - Ask for help with problems] [answered]
处理 function ale#events#LintOnEnter[5]..ale#Queue[33]..<SNR>57_Lint[20]..ale#engine#RunLinters[4]..<SNR>66_GetL
eValues[27]..<lambda>11[1]..<SNR>66_RunLinters[12]..<SNR>66_RunLinter[2]..ale#lsp_linter#CheckWithLSP[1]..ale#ls
er#StartLSP[3]..ale#lsp_linter#FindProjectRoot 时发生错误:
第   22 行:
E121: 未定义的变量: g:ale_root

Please help me thx

**Comment by hsanson** [comment, marked as answer]
This similar issue may help: https://github.com/dense-analysis/ale/issues/3626

**Reply by Alice-pleasance-liddell** [reply]
thx， it's not appear again
